tests/phpunit/elementor/modules/dev-tools/test-deprecation.php

```diff
@@ -205,4 +205,36 @@ public function test_deprecated_function_hard() {
 
 		$this->deprecation->deprecated_function( __FUNCTION__, '0.0.0', '', '0.5.0' );
 	}
+
+	public function test_do_deprecated_action() {
+		$this->setExpectedDeprecated( 'elementor/test/deprecated_action' );
+
+		add_action( 'elementor/test/deprecated_action', function() {
+			echo 'Testing Do Deprecated Action';
+		} );
+
+		ob_start();
+
+		$this->deprecation->do_deprecated_action( 'elementor/test/deprecated_action', [], '0.0.0', '', '0.5.0' );
+
+		$result = ob_get_clean();
+
+		$this->assertEquals( $result, 'Testing Do Deprecated Action' );
+	}
+
+	public function test_do_deprecated_action_soft() {
```

Comment:
```suggestion
	public function test_do_deprecated__action_soft() {
```

Reply:
why?

Reply:
I fixed it, it the method name should be separated from the use case.

Reply:
I Don't understand @matipojo

Reply:
Changed it to double underscore.